boa_cli/src/main.rs

```diff
@@ -154,6 +156,14 @@ struct Opt {
     /// Inject debugging object `$boa`.
     #[arg(long)]
     debug_object: bool,
+
+    /// Treats the input files as modules.
+    #[arg(long, short = 'm', group = "mod")]
+    module: bool,
+
+    /// Root path from where the module resolver will try to load the modules.
+    #[arg(long, default_value_os_t = PathBuf::from("."), requires = "mod")]
```

Comment:
Maybe `root` instead of `modpath`. That would bring it more in line with `SingleModuleLoader` and a short `r` could be used as well.